chapters/packages.tex

```diff
@@ -323,6 +323,7 @@ \section{External Resources}\label{external-resources}
 \end{example}
 
 \section{Multilingual Descriptions}\label{multilingual-descriptions}
+\index{translation}
```

Comment:
Usually a good idea to take measures to avoid unwanted space resulting from the newline at the end of the first line:
```suggestion
\section{Multilingual Descriptions}\label{multilingual-descriptions}\index{translation}
```